http_logs/track.py

```diff
@@ -7,10 +13,38 @@ async def reindex_async(es, params):
     result = await es.reindex(body=params.get("body"), request_timeout=params.get("request_timeout"))
     return result["total"], "docs"
 
+class RuntimeFieldResolver(loader.TrackProcessor):
+    PATTERN = re.compile('.+-from-(.+)-using-(.+)')
+
+    def on_after_load_track(self, t):
+        for challenge in t.challenges:
+            for task in challenge.schedule:
+                m = self.PATTERN.match(task.name)
+                if m is not None:
+                    print(f"ASDF {task.name} {task.operation} {task.operation.params}")
+                    task.operation = copy(task.operation)
+                    task.operation.params = self._replace_field(f"{m[2]}.from_{m[1]}.", task.operation.params)
+                    print(f"ASDF {task.name} {task.operation} {task.operation.params}")
```

Comment:
These are debugging leftovers but I think they are useful to see the changes it makes. I figure a reviewer would want to see this output anyway. So I'll remove later.